.travis.yml

```diff
@@ -46,7 +47,7 @@ before_deploy:
   - echo "$REGISTRY_PASS" | docker login --username "$REGISTRY_USER" --password-stdin
 deploy:
   provider: script
-  script: dcd --tag $TAGS --version $version --version-semver ${VERSION_LATEST:+--version-latest} "$image_tag"
+  script: dcd -t $TAGS -v $version -S ${VERSION_LATEST:+-L} -P ${SEMVER_PRECISION:-1} "$image_tag"
```

Comment:
Can we maybe use the long forms of `-S`, `-L`, and `-P`? `-t` and `-v` are fairly clear from context, but the others aren't.